src/instructlab/model/accelerated_train.py

```diff
@@ -136,6 +154,9 @@ def accelerated_train(
                 bg="yellow",
                 fg="black",
             )
+            # Start at phase 2 for skills only training
+            if strategy == SupportedTrainingStrategies.LAB_SKILLS_ONLY.value:
```

Comment:
I guess this is the crux of 'skipping phase 1' which worked out nicely

Reply:
Correct.  Or at least that's what I worked out as a reasonable approach.  And agreed it proves out the journal approach was well thought out if this is indeed the appropriate change.